app/controllers/students_controller.rb

```diff
@@ -2,4 +2,37 @@ class StudentsController < ApplicationController
   def index
     @students = Student.paginate(page: params[:page], per_page: 10)
```

Comment:
I understand this is outside the scope of this PR, but I spot a magic number!

I recommend moving the `per_page` value into a constant e.g.

```ruby
class StudentsController < ApplicationController
  DEFAULT_PER_PAGE = 10

  def index
    @students = Student.paginate(page: params[:page], per_page: DEFAULT_PER_PAGE)
  end
```